andela/savage_rangers-ah
# #166240819 email verification

#### What does this PR do?
 * Enable the user to verify email.
#### Description of Task to be completed?
 * After successful registration, an email is sent to the user's personal email provider.
 * Once the user receives the email with the verification link and clicks it, the user is redirected to the ``/auth/verifyEmail`` endpoint to verify the user.
#### How should this be manually tested?
* Clone the repo to  local machine.
* Run ``npm install``   to install the dependencies.
* Set up the   ``.env`` as instructed by the   ``.env-example``
* Run  ``npm  start``.
* Post registration data to  ``/api/auth/signup``   and check your email (check the spam).
*  Click the verification link.
#### What are the relevant pivotal tracker stories?
#[166240819](https://www.pivotaltracker.com/story/show/166240819)
#### Screenshots (if appropriate)
<img width="1440" alt="Screen Shot 2019-06-13 at 18 07 18" src="https://user-images.githubusercontent.com/37019531/59452872-fa356380-8e0e-11e9-9343-d00ccbc17f35.png">
<img width="1440" alt="Screen Shot 2019-06-13 at 19 04 11" src="https://user-images.githubusercontent.com/37019531/59452896-08837f80-8e0f-11e9-81c7-51d7a36010b9.png">
<img width="1440" alt="Screen Shot 2019-06-13 at 19 04 47" src="https://user-images.githubusercontent.com/37019531/59453023-4a142a80-8e0f-11e9-9986-2a5bed9471c8.png">
